src/components/routes/Starred.vue

```diff
@@ -42,6 +68,14 @@ export default defineComponent({
 		oldestFirst() {
 			return this.$store.getters.oldestFirst
 		},
+
+		id(): number {
+			return Number(this.feedId)
+		},
+
+		fetchKey(): string {
+			return (this.feedId === 0) ? 'starred' : 'starred-' + this.feedId
+		},
```

Comment:
```suggestion

		id(): number {
			return this.feedId ? Number(this.feedId) : 0
		},

		fetchKey(): string {
			return this.feedId ? 'starred-' + this.feedId : 'starred'
		},
```
Handle undefined feedId string.